src/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+const store = configureStore({
+  reducer: {},
```

Comment:
- [ ] it would be better to add the exported reducers in the reducers in order to be ready to fetch and add books to the store so please try to add books and categories reducers here